src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2669,11 +2669,13 @@ static void _inheritGradient(SvgLoaderData* loader, SvgStyleGradient* to, SvgSty
         if (to->transform) memcpy(to->transform, from->transform, sizeof(Matrix));
     }
 
-    if (to->type == SvgGradientType::Linear && from->type == SvgGradientType::Linear) {
```

Comment:
This condition looks like somewhat unnecessary changes. It doesn't need to loop for().

Reply:
from and to can be different gradients(linear, radial).
Regardless of what the from is, recalc must be performed according to changes in common properties.(unit, transform, spread)

Reply:
@JSUYA please see 2675 line.

Reply:
ok I removed 2675line if condition